src/renderer/components/publication/Cover.tsx

```diff
@@ -44,16 +35,24 @@ export default class Cover extends React.Component<ICoverProps, null> {
                 }
                 authors += this.translator.translateContentField(newAuthor);
             }
+            let colors = this.props.publication.customCover;
+            if (colors === undefined) {
+                colors = {
+                    topColor: "#d18e4b",
+                    bottomColor: "#7c4c1c",
+                };
```

Comment:
Instead of hard-coding the CSS colour strings here, would it be possible to reference them from the existing `common/models/custom-cover.ts` file?
https://github.com/readium/readium-desktop/blob/cd3997f902be46d4299dc97e0f3ccbdd09fbe69c/src/common/models/custom-cover.ts#L13-L26

Once centralized, these hard-coded CSS styles will be easier to refactor / extract into proper CSS variables (or any other applicable technique), see related issue https://github.com/readium/readium-desktop/issues/409